src/datasource.js

```diff
@@ -30,6 +30,18 @@ export class GenericDatasource {
       query.adhocFilters = [];
     }
 
+    const index = _.isUndefined(this.templateSrv.index) ? {} : this.templateSrv.index;
```

Comment:
this is not allowed, not sure what your trying to do here and what the point of doing this is

Reply:
This is fetching all set variables so they can be sent to API endpoint